core-bundle/src/Resources/config/services_debug.yml

```diff
@@ -0,0 +1,5 @@
+services:
+    contao.debug.security.access.decision_manager:
+        class: Symfony\Component\Security\Core\Authorization\TraceableAccessDecisionManager
+        arguments:
```

Comment:
doesn't this miss the service to decorate?

Reply:
Lol, of course! I tested, then noticed that I forgot the `decorates` info, added this but then forgot to commit that :D Fixed now, thanks!